site/app/controllers/GlobalController.php

```diff
@@ -403,6 +403,14 @@ public function footer() {
                 }
             }
         }
+        // append the help links
+        if ($this->core->getConfig()->getSysAdminUrl() != '') {
+            $footer_links[] =  ["title" => "Report Issues", "url" => $this->core->getConfig()->getSysAdminUrl()];
+        }
+        if ($this->core->getConfig()->getSysAdminEmail() != '') {
+            $footer_links[] =  ["title" => "Email Sys. Admin", "url" => $this->core->getConfig()->getSysAdminEmail(), "isEmai" => true];
```

Comment:
typo here for `isEmai` (should be `isEmail`?)

Reply:
Done